.env.example

```
@@ -14,3 +14,4 @@ AMI_NUM_WORKERS=1
 AMI_ANTENNA_API_BASE_URL=http://localhost:8000/api/v2
 AMI_ANTENNA_API_AUTH_TOKEN=your_antenna_auth_token_here
 AMI_ANTENNA_API_BATCH_SIZE=4
+AMI_ANTENNA_SERVICE_NAME=AMI Data Companion
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)